docs/uefi.md

```diff
@@ -35,8 +35,41 @@ To make Cloud Hypervisor use UEFI boot, pass the `OVMF.fd` file path as an argum
 
 The same firmware can be used with Cloud Hypervisor or with QEMU. This is particularly useful if using QEMU for the preparation phase.
 
+## Building UEFI Firmware with Compatibility Support Module (CSM)
+
+CSM is a module that allows to boot legacy operating systems using the OVMF firmware. OVMF can embed a CSM build of SeaBIOS. To build the SeaBIOS with CSM support, add `CONFIG_CSM=y` to `.config` before the build. The outcome `out/Csm16.bin` is to be moved into `OvmfPkg/Csm/Csm16/Csm16.bin` before OVMF is built. Also, the OVMF will need `-D CSM_ENABLE` to be appended, which will produce a legacy aware UEFI firmware. At the current stage, all the necessary patches are included in the Cloud Hypervisor specific [SeaBIOS branch](https://github.com/cloud-hypervisor/seabios/tree/ch). Taking into account the previous instructions, the modified command sequence to compile UEFI with the CSM support is below:
```

Comment:
s/`Also, the OVMF will need `-D CSM_ENABLE` to be appended,`/`Then, the OVMF build will have to be passed the  `-D CSM_ENABLE` option in order to generate a legacy aware UEFI firmware,`

Reply:
Applied, thanks.